app/models/planet.py

```diff
@@ -0,0 +1,15 @@
+from app import db
+
+class Planet(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String, nullable = False)
+    description = db.Column(db.String, nullable = False)
+    size = db.Column(db.String, nullable = False)
```

Comment:
Size could certainly be a string, but perhaps there's another data type that might work better?